semgrep/semgrep/output.py

```diff
@@ -321,7 +324,7 @@ def output(
                     suggestion_line = "(need more rules? `semgrep login` for additional free Semgrep Registry rules)\n"
                 else:
                     suggestion_line = ""
-                stats_line = f"Ran {unit_str(num_rules, 'rule')} on {unit_str(num_targets, 'file')}: {unit_str(num_findings, 'finding')}."
+                stats_line = f"ran {num_rules} rules on {num_targets} files: {num_findings} findings"
```

Comment:
```suggestion
               f"Ran {unit_str(num_rules, 'rule')} on {unit_str(num_targets, 'file')}: {unit_str(num_findings, 'finding')}."
```
I'm assuming this was an unintended regression given you made the change on the left in a previous PR

Reply:
I'm gonna commit this and merge so we can get this in for release. :D open another PR with this change if i'm mistaken